### PR TITLE
Contact form validation

### PR DESCRIPTION
**Pull request by looh16**
Add a client-side validation of the data in the contact form before submission.

### CODE REVIEW

File: js/form-validate.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const email = document.querySelector('input[name=user_email]');
+const button = document.querySelector('#btn-get-touch');
+const text = document.querySelector('#message');
+
+const validateEmail = (email) => {
+  /* eslint-disable no-useless-escape */
+  const regex = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return regex.test(String(email).toLowerCase());
```
Comment (Ginohmk) [inline js/form-validate.js:8]: - Good job on your form validation, but there are some requirements missing which are.
  -  The form those not submitted when a valid email is entered
  - The error message should show up close to the "submit" button.
**Please implement the requirements above**  Happy coding :+1: 

- [Optional] The form design is looking different from that of the Figma design, please try to fix it so your portfolio looks professional.

#####  Video of Your design test run

https://user-images.githubusercontent.com/58771507/175769417-fc3282e6-dfcb-41e5-ba19-7adde312cc23.mp4

**Review by Ginohmk — CHANGES_REQUESTED** [review]
Hi @looh16 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- No linter errors :+1: 
- Weel organized file structure :+1: 
- Good commit messages :+1: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can reach this Code reviewer on slack:
- `@kanu mike Chibundu`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/form-validate.js (hunk 1 above)
Comment (looh16) [reply js/form-validate.js:8]: Thank you! I will include the recommendations, just point out that the exercise only refers to the email field.

File: js/form-validate.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+};
+
+const validateLowerCaseEmail = (email) => {
+  /* eslint-disable no-useless-escape */
+  const regex = /^([a-z\d\.-]+)@([a-z\d-]+)\.([a-z]{2,8})(\.[a-z]{2,8})?$/;
+  return regex.test(email);
```
Comment (Ismailco) [inline js/form-validate.js:14]: - [ ] [OPTIONAL] This project should use peer-programming, Kindly make sure all the students in the group contribute to this PR

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @looh16,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kandy-Peter — APPROVED** [review]
Hi team,

### APPROVED 🟢 

Great implementation of all the requirements in this milestone. 👏
Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

##  Optional suggestion

# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**